fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1799,4 +1799,10 @@ public class Config extends ConfigBase {
      **/
     @ConfField(mutable = true)
     public static int privilege_max_role_depth = 16;
+
+    /**
+     * ignore invalid privilege & authentication when upgraded to new RBAC privilege framework in 3.0
+     */
+    @ConfField(mutable = true)
+    public static boolean ignore_invalid_privilege_authentications = false;
```

Comment:
in what circumstance do we need to open this switch ?

Reply:
If we have confirmed that all invalid entries can be ignored.